source/engine/graphics2/diligent/DiligentEngineWin32.cpp

```diff
@@ -0,0 +1,170 @@
+#include "DiligentEngine.h"
+#include "ncengine/utility/Log.h"
+#include "ncengine/window/Window.h"
+#include "ncutility/NcError.h"
+
+#define GLFW_EXPOSE_NATIVE_WIN32 1
+#include "Graphics/GraphicsEngineD3D11/interface/EngineFactoryD3D11.h"
```

Comment:
These used to be wrapped in an #if PLATFORM_WIN32. I'll move these down to the other Diligent headers